.github/workflows/go.yml

```
@@ -0,0 +1,24 @@
+name: Go
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
```

It's better to allow pull request on any branch, and double quotes are not required in here:
```suggestion
  push:
    branches: [master]
  pull_request:
```

---

updated